openedx/core/djangoapps/user_authn/views/tests/test_logistration.py

```diff
@@ -1,334 +1,41 @@
-        """
-        # Log out the user created during test setup, to prevent the view from
-        # selecting the logged-in user's email address over the email provided
-        # in the POST data
-        self.client.logout()
-        for status in [200, 403]:
-            response = self._change_password(email=self.NEW_EMAIL)
-            self.assertEqual(response.status_code, status)
-
-        with mock.patch(
-            'util.request_rate_limiter.PasswordResetEmailRateLimiter.is_rate_limit_exceeded',
-            return_value=False
-        ):
-            response = self._change_password(email=self.NEW_EMAIL)
-            self.assertEqual(response.status_code, 200)
-
-    @ddt.data(
-        ('post', 'password_change_request', []),
-    )
-    @ddt.unpack
-    def test_require_http_method(self, correct_method, url_name, args):
-        wrong_methods = {'get', 'put', 'post', 'head', 'options', 'delete'} - {correct_method}
-        url = reverse(url_name, args=args)
-
-        for method in wrong_methods:
-            response = getattr(self.client, method)(url)
-            self.assertEqual(response.status_code, 405)
-
-    def _change_password(self, email=None):
-        """Request to change the user's password. """
-        data = {}
-
-        if email:
-            data['email'] = email
-
-        return self.client.post(path=reverse('password_change_request'), data=data)
-
-    def _create_dop_tokens(self, user=None):
-        """Create dop access token for given user if user provided else for default user."""
-        if not user:
-            user = User.objects.get(email=self.OLD_EMAIL)
-
-        client = ClientFactory()
-        access_token = AccessTokenFactory(user=user, client=client)
-        RefreshTokenFactory(user=user, client=client, access_token=access_token)
-
-    def _create_dot_tokens(self, user=None):
-        """Create dop access token for given user if user provided else for default user."""
-        if not user:
-            user = User.objects.get(email=self.OLD_EMAIL)
-
-        application = dot_factories.ApplicationFactory(user=user)
-        access_token = dot_factories.AccessTokenFactory(user=user, application=application)
-        dot_factories.RefreshTokenFactory(user=user, application=application, access_token=access_token)
-
-    def assert_access_token_destroyed(self, user):
-        """Assert all access tokens are destroyed."""
-        self.assertFalse(dot_access_token.objects.filter(user=user).exists())
-        self.assertFalse(dot_refresh_token.objects.filter(user=user).exists())
-        self.assertFalse(dop_access_token.objects.filter(user=user).exists())
-        self.assertFalse(dop_refresh_token.objects.filter(user=user).exists())
-
 
 @skip_unless_lms
 @ddt.ddt
 class LoginAndRegistrationTest(ThirdPartyAuthTestMixin, UrlResetMixin, ModuleStoreTestCase):
-    """ Tests for the student account views that update the user's account information. """
```

Comment:
Possible clean-up for a follow-up PR:
When looking at the file, I noticed the following `assertNotContains`: https://github.com/edx/edx-platform/blob/63a1a1db97e857d28f708117239440eb388c8758/openedx/core/djangoapps/user_authn/views/tests/test_logistration.py#L602
We had some tests that I fixed that used `assertNotContains`, but no parallel test with `assertContains`, and the thing that wasn't being found was because it was bogus code, not because some flag was working appropriately.